test/tools/Modules/HelpersCommon/HelpersCommon.psm1

```diff
@@ -11,14 +11,35 @@ function Wait-UntilTrue
 
     # Loop until the script block evaluates to true
     while (-not ($sb.Invoke())) {
+        # If the timeout period has passed, return false
+        if (([DateTime]::Now - $startTime).TotalMilliseconds -gt $timeoutInMilliseconds) {
+            return $false
+        }
         # Sleep for the specified interval
-        start-sleep -mil $intervalInMilliseconds
+        Start-Sleep -Milliseconds $intervalInMilliseconds
+    }
+    return $true
+}
 
-        # If the timeout period has passed, throw an exception
-        if (([DateTime]::Now - $startTime).TotalMilliseconds -gt $timeoutInMilliseconds)
-        {
+function Wait-FileToBePresent
+{
+    [CmdletBinding()]
+    param (
+        [string]$File,
+        [int]$TimeoutInSeconds = 10,
+        [int]$IntervalInMilliseconds = 100
+    )
+    # Get the current time
+    $startTime = [DateTime]::Now
+
+    # Loop until the file is present
+    while (-not (Test-Path $File)) {
```

Comment:
Maybe use `Wait-UntilTrue`? We duplicate code here.

Reply:
Good point. I will change it.